libs/sdk-ui-pivot/src/impl/drilling/drillIntersectionFactory.ts

```diff
@@ -27,13 +34,23 @@ export function createDrillIntersection(
     const col: AnyCol = tableDescriptor.getCol(cellEvent.colDef);
     const row = cellEvent.data as IGridRow;
 
-    invariant(isSliceCol(col) || isSeriesCol(col) || (tableDescriptor.isTransposed() && isScopeCol(col)));
+    invariant(
```

Comment:
reuse invariant from onCellClick in form of some shared helper?

Reply:
Resolving as discussed offline, issues with type guard checks.